src/metatensor/models/experimental/pet/train.py

```diff
@@ -25,12 +25,15 @@ def train(
     hypers: Dict = DEFAULT_HYPERS,
     continue_from: Optional[str] = None,
     output_dir: str = ".",
-    device_str: str = "cpu",
+    devices: List[torch.device] = [torch.device("cpu")],  # noqa: B006, B008
```

Comment:
this does not match the `DEVICE=['cuda']` in `__init__.py`

Reply:
These defualt devices are now gone